components/11textut.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style.css">
+    <link rel="stylesheet" href="/docs.css">
+    <title>Even UI | Texts</title>
+</head>
+
+<body>
+    <nav>
+        <div class="nav-bar">
+            <ul class="nav-list">
+                <li class="nav-items"><img class="nav-icon" src="/images/undraw_nature_m5ll.svg" alt=""></li>
+                <a href="/index.html">
+                    <li class="nav-items">Home</li>
+                </a>
+            
+            </ul>
+            <ul class="nav-list">
+                <a href="https://github.com/h-harsh"> <li class="nav-items">Github</li></a>
+            </ul>
+        </div>
+    </nav>
+    <main>
+        <div class="sidebar">
+            <ul>
+                <a href="/components/1avatar.html">
+                    <li class="sidebar-item">Avatars</li>
+                </a>
+                <a href="/components/2badge.html">
+                    <li class="sidebar-item">Badge</li>
+                </a>
+                <a href="/components/3alerts.html">
+                    <li class="sidebar-item">Alert</li>
+                </a>
+                <a href="/components/4button.html">
+                    <li class="sidebar-item">Buttons</li>
+                </a>
+                <a href="/components/5cards.html">
+                    <li class="sidebar-item">Cards</li>
+                </a>
+                <a href="/components/6image.html">
+                    <li class="sidebar-item"> Image</li>
+                </a>
+                <a href="/components/7input.html">
+                    <li class="sidebar-item">Input Fields</li>
+                </a>
+                <a href="/components/8lists.html">
+                    <li class="sidebar-item">Lists</li>
+                </a>
+                <a href="/components/9navbar.html">
+                    <li class="sidebar-item">Nav Bar</li>
+                </a>
+                <a href="/components/10grid.html">
+                    <li class="sidebar-item">Grid</li>
+                </a>
+                <a href="/components/11textut.html">
+                    <li class="sidebar-item">Text Utilities</li>
+                </a>
+            </ul>
+        </div>
+        <div class="components">
+            <div class="sub-comp-div">
+                <h2 class="div-ch">Headings</h2>
+                <div class="div-ch">
+                    <h1># Heading one</h1>
+                    <h2>## Headng two</h2>
+                    <h3>### Heading three</h3>
+                    <h4>#### Heading four</h4>
+                    <h5>##### Heading five</h5>
+                    <h6>###### Heading six</h6>
+                    <p>This is paragraph text</p>
+                </div>
```

Comment:
Give some margin and padding to each content

Reply:
Done